Module/itp/waypointpalcus.itp.json

```diff
@@ -413,17 +413,6 @@
                       "value": "womprat_spwn"
                     }
                   },
-                  {
```

Comment:
I don't think this is intentional either. You're removing the new stuff Valo added. Make sure your copy of the module is up to date.